js/test/Ice/properties/Client.js

```diff
@@ -50,29 +50,6 @@ export class Client extends TestHelper {
             for (const [key, value] of props) {
                 test(properties.getProperty(key) == value);
             }
-        } else if (typeof window !== "undefined") {
-            /* eslint-enable no-sync */
```

Comment:
This was using jQuery ajax, maybe I should port it to use `fetch` API? But seems parse is already tested with NodeJS and shouldn't be a difference with browsers.

Reply:
Yeah I wouldn't bother.